willow/plugins/pillow.py

```diff
@@ -166,34 +166,40 @@ def set_background_color_rgb(self, color):
         return PillowImage(new_image.convert("RGB"))
 
     @Image.operation
-    def save_as_jpeg(self, f, quality=85, optimize=False, progressive=False):
+    def save_as_jpeg(
+        self, f, quality=85, optimize=False, progressive=False, apply_optimizers=True
+    ):
         if self.image.mode in ["1", "P"]:
             image = self.image.convert("RGB")
         else:
             image = self.image
 
         # Pillow only checks presence of optimize kwarg, not its value
```

Comment:
This comment feels a touch misplaced now... I wonder if it's outstayed it's welcome?